docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.9"
+services:
+  bash:
+    image: brew-control-dash
+    ports:
+      - "5000:5000"
```

Comment:
is there a reason we need this port exposed for this service? typically i'd only expect e.g. the flask app service to need a port